modules/microshift-uninstalling-lvms-csi-snapshot.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-uninstalling-lvms-csi-snapshot_{context}"]
+== Uninstalling the CSI driver and CSI snapshot implementation
```

Comment:
hey, I just wanted to check - in https://github.com/openshift/openshift-docs/pull/80823 I would rather use "disable" than "make optional" - as by default it is installed (or should be by default) - so the naming should be
Making LVMS and CSI volume snapshot deployment optional -> Disable LVMS and CSI volume snapshot deployment

(comment from @vwalek)

Reply:
now the heading is "LVMS CSI provider and CSI snapshot deployments". I can change it to "Disable LVMS CSI provider and CSI snapshot deployments" to include suggestion

Reply:
resolving as per discussion on the heading